<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="W0">
   <layout class="QVBoxLayout" name="L0">
    <item>
     <widget class="QWidget" name="W1" native="true">
      <layout class="QHBoxLayout" name="L1">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="W2" native="true">
         <layout class="QVBoxLayout" name="L2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="logo">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>186</width>
                 <height>240</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="mainwindow.qrc">:/Sxkq4ox.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCameraViewfinder" name="viewFinder" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>320</width>
                 <height>240</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="W3" native="true">
            <layout class="QVBoxLayout" name="L4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="widget" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="startButton">
                  <property name="font">
                   <font>
                    <pointsize>26</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Start Validation</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="cancelButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>26</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Cancel Validation</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="text_label">
               <property name="font">
                <font>
                 <pointsize>26</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Validation Status: &lt;font color='red'&gt;None&lt;/font&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="W4" native="true">
               <layout class="QHBoxLayout" name="L5">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="total_price">
                  <property name="font">
                   <font>
                    <pointsize>26</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Amount To Charge:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="total_price_edit">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>26</pointsize>
                   </font>
                  </property>
                  <property name="placeholderText">
                   <string>$0.00</string>
                  </property>
                  <property name="clearButtonEnabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QCommandLinkButton" name="pay_button">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="font">
                <font>
                 <pointsize>26</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Pay Now</string>
               </property>
               <property name="icon">
                <iconset resource="mainwindow.qrc">
                 <normaloff>:/FullResSquareLogo.png</normaloff>:/FullResSquareLogo.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="console">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::NoWrap</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="tabStopWidth">
       <number>80</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCameraViewfinder</class>
   <extends>QWidget</extends>
   <header>qcameraviewfinder.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
